src/components/FieldLabel/FieldLabel.tsx

```diff
@@ -15,13 +20,35 @@ interface FieldLabelProps extends VibeComponentProps {
 
 const FieldLabel: FC<FieldLabelProps> = forwardRef(
   (
-    { icon = "", iconLabel = "", labelText = "", labelFor = "", iconClassName = "", labelClassName = "" },
+    {
+      icon = "",
+      descriptionIcon = undefined,
+      description = "",
+      iconLabel = "",
+      labelText = "",
+      labelFor = "",
+      iconClassName = "",
+      labelClassName = ""
+    },
     ref: ForwardedRef<HTMLLabelElement>
   ) => {
     if (!labelText) {
       return null;
     }
 
+    const renderIconWithToolTip = () => {
+      return (
+        <Tooltip content={description}>
+          <Icon
+            icon={descriptionIcon ?? Info}
+            clickable={false}
+            iconType={Icon.type.ICON_FONT}
```

Comment:
I think the icon type is SVG and not font, I know that other icons here are with type font but right now our best practice is to use SVG icons